_posts/2025-01-23-ad-rule-cx-success.md

```diff
@@ -0,0 +1,98 @@
+---
+layout: post
+
+title:  "Enhancing anomaly detection in Amazon OpenSearch Service: A customer success story"
+authors:
+ - kaituo
+ - kentonparton
+ - amitgalitz
+date: 2025-01-22
+categories:
+ - technical-posts
+ - community
+meta_keywords: OpenSearch anomaly detection, rule, expected value, ratio, customer success
```

Comment:
Updated meta: 

_meta_keywords:_ OpenSearch anomaly detection, EBSCO, anomaly detection plugin, configure anomaly detection rules, logging system monitoring

_meta_description:_ Learn how Amazon OpenSearch Service's enhanced anomaly detection helped EBSCO reduce false positives and improve system monitoring through new rule-based detection capabilities. Explore practical implementations of these features and their impact on cloud service reliability.

Reply:
updated